Docs/starters-version-control-formstack-themes

clairesunstudio opened the pull request:
Move formstack themes into this repo for version control and consolidation 

To-do:
- consolidate the themes into one 
  currently being used in prod
   1. mayflower
   2. mayflower standalone
   3. mayflower without button
- Cleanup css and convert it to SCSS to ingest Mayflower variables and mixins
- Add a formstack template markup to include all the fields for testing (potentially through API so that it's also up-to-date with Formstack UI)

clairesunstudio committed:
add formstack theme

clairesunstudio committed:
add formstack theme

clairesunstudio committed:
update to noto sans

clairesunstudio committed:
update to noto sans

clairesunstudio committed:
copy of most up-to-date theme

clairesunstudio committed:
copy of most up-to-date theme

clairesunstudio committed:
add standalone theme

clairesunstudio committed:
add standalone theme

tkoleary commented:
Looks good!

tkoleary reviewed (approved):
Looks good to me. I have some follow-ups of things I saw after creating a test form but I'll start a new ticket for those.

clairesunstudio committed:
Merge branch 'develop' into docs/add-starters

tkoleary committed:
Docs/more-formstack-updates (#1297)

* add formstack theme

* update to noto sans

* copy of most up-to-date theme

* add standalone theme

* Updates to mayflower formstack theme

-grouped checkboxess/radios together
-made "other" option full width
-added selector for vertical checkboes and radios
-fixed layout of credit card number
-made matrix width  100%
-fixed ssome errors caught by formstack linter
-added prefixes for transition and transform
-added comments to css

* Update mayflower-theme.css

Co-authored-by: Minghua Sun <clairesunstudio@gmail.com>

tkoleary committed:
Docs/more-formstack-updates (#1297)

* add formstack theme

* update to noto sans

* copy of most up-to-date theme

* add standalone theme

* Updates to mayflower formstack theme

-grouped checkboxess/radios together
-made "other" option full width
-added selector for vertical checkboes and radios
-fixed layout of credit card number
-made matrix width  100%
-fixed ssome errors caught by formstack linter
-added prefixes for transition and transform
-added comments to css

* Update mayflower-theme.css

Co-authored-by: Minghua Sun <clairesunstudio@gmail.com>

tkoleary committed:
Merge branch 'docs/add-starters' of https://github.com/massgov/mayflower into docs/add-starters

clairesunstudio committed:
Merge branch 'develop' into docs/add-starters
